js/coinflex.js

```diff
@@ -0,0 +1,297 @@
+                        'v2/repay/{asset}': 1,
+                        'v2/borrowingSummary': 1,
+                        'v2/candles': 1,
+                        'auth/self/verify': 1,
+                        'borrow/': 1,
+                        'repay/': 1,
+                        'borrowingSummary': 1,
+                        'funding-payments': 1,
+                        'AMM': 1,
+                    },
+                    'post': {
+                        'v2.1/delivery/orders': 1,
+                        'v2/orders/place': 1,
+                        'v2/orders/modify': 1,
+                        'v2/mint': 1,
+                        'v2/redeem': 1,
+                        'v2/borrow': 1,
+                        'v2/repay': 1,
+                        'v2/borrow/close': 1,
+                        'v2/AMM/create': 1,
+                        'v2/AMM/redeem': 1,
+                        'v3/withdrawal': 1,
+                        'v3/transfer': 1,
+                        'v3/flexasset/mint': 1,
+                        'v3/flexasset/redeem': 1,
+                        'v3/AMM/create': 1,
+                        'v3/AMM/redeem': 1,
+                    },
+                    'delete': {
+                        'v2/cancel/orders': 1,
+                        'v2/cancel/orders/{marketCode}': 1,
+                        'v2.1/delivery/orders/{deliveryOrderId}': 1,
+                        'v2/orders/cancel': 1,
+                    },
+                },
+                'private': {
+                },
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': this.parseNumber ('0.000'),
+                    'taker': this.parseNumber ('0.008'),
+                },
+            },
+            'options': {
+                'timeframes': {
+                    'spot': {
+                    },
+                    'contract': {
+                    },
+                },
+                'defaultType': 'spot', // spot, swap
+                'networks': {
+                    'TRX': 'TRC-20',
+                    'TRC20': 'TRC-20',
+                    'ETH': 'ERC-20',
+                    'ERC20': 'ERC-20',
+                    'BEP20': 'BEP-20(BSC)',
+                },
+            },
+            'commonCurrencies': {
+            },
+            'exceptions': {
+                'exact': {
+                },
+                'broad': {
+                },
+            },
+        });
+    }
+
+    async fetchMarkets (params = {}) {
+        const response = await this.publicGetV3Markets (params);
+        
+    }
+
+    sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
+        [ path, params ] = this.resolvePath (path, params);
+        let url = this.urls['api'][api] + '/' + path;
+        let paramsSortedEncoded = '';
```

Comment:
We don't need this variable yet, as long as we encode them in-place for now anyway. In short we almost never initialize a variable to an empty string.